tasks.py

```diff
@@ -1530,7 +1530,7 @@ def fetch_kubectl():
     curl_command = "curl -o {} -LO https://dl.k8s.io/release/{}/bin/$(go env GOOS)/$(go env GOARCH)/kubectl".format(
         kubectl_path, kubectl_version
     )
-    get_version_command = f"{kubectl_path} version --short"
+    get_version_command = f"{kubectl_path} version"
```

Comment:
```
 kubectl version
Client Version: v1.31.0
Kustomize Version: v5.4.2
Server Version: v1.31.0
```
we probably need to isolate only the Client Version

Reply:
I think what you're asking is already done in https://github.com/metallb/metallb/blob/17aafdea571b097012fafe758d921a388fff9f46/tasks.py#L1538-L1539